### ISSUE TITLE
Implement gradient clipping in Fluid

### ISSUE THREAD

**Issue by jacquesqiao** [PaddlePaddle/Paddle#6447]
Gradient Clipping is a useful trick for training a very deep neural network.
There could be many kinds of clipping methods:

1. Clipping by a value directly
2. Clipping by a global norm.

**Comment by qingqing01** [comment]
Just for reminding, the [clipping operator](https://github.com/PaddlePaddle/Paddle/blob/develop/paddle/operators/clip_op.h) and [clip_ by_norm operator](https://github.com/PaddlePaddle/Paddle/blob/develop/paddle/operators/clip_by_norm_op.h) have been implemented, these are needed to import into Python API.

**Comment by jacquesqiao** [comment]
Done by @reyoung
